noxfile.py

```diff
@@ -212,6 +212,7 @@ def lint(session):
         ".git",
         "website",
         "plugins",
+        "__init__.py",
```

Comment:
ditto

Reply:
I think I'd have to exclude it `__init__.py` hear as well